src/node_contextify.cc

```diff
@@ -745,6 +747,26 @@ class ContextifyScript : public BaseObject {
   }
 
 
+  static void CreateCachedData(const FunctionCallbackInfo<Value>& args) {
+    Environment* env = Environment::GetCurrent(args);
+    ContextifyScript* wrapped_script;
+    ASSIGN_OR_RETURN_UNWRAP(&wrapped_script, args.Holder());
+    Local<UnboundScript> unbound_script =
+        PersistentToLocal(env->isolate(), wrapped_script->script_);
+    std::unique_ptr<ScriptCompiler::CachedData> cached_data(
+        ScriptCompiler::CreateCodeCache(unbound_script));
+    if (!cached_data) {
+      THROW_ERR_CREATE_CACHED_DATA_FAILED(env, "Creating cached data failed.");
+    } else {
+      MaybeLocal<Object> buf = Buffer::Copy(
+          env,
+          reinterpret_cast<const char*>(cached_data->data),
+          cached_data->length);
+      args.GetReturnValue().Set(buf.ToLocalChecked());
```

Comment:
It may be a good idea to throw `ERR_BUFFER_TOO_LARGE` if `cached_data->length > Buffer::kMaxLength` before `Buffer::Copy` and `ERR_MEMORY_ALLOCATION_FAILED` if `buf.IsEmpty()` is true